starknet_py/net/account/account_client.py

```diff
@@ -307,6 +316,74 @@ async def sign_transaction(
 
         return execute_tx
 
+    async def sign_transaction(
+        self,
+        calls: Calls,
+        max_fee: Optional[int] = None,
+        auto_estimate: bool = False,
+        version: int = 0,
+    ) -> InvokeFunction:
+        """
+        Takes calls and creates signed InvokeFunction
+
+        :param calls: Single call or list of calls
+        :param max_fee: Max amount of Wei to be paid when executing transaction
+        :param auto_estimate: Use automatic fee estimation, not recommend as it may lead to high costs
+        :param version: Transaction version
+        :return: InvokeFunction created from the calls
+
+        .. deprecated:: 0.4.6
+            This method has been replaced by :meth:`AccountClient.create_invoke_transaction`
+        """
+        # TODO add correct deprecation version
```

Comment:
what about this?

Reply:
This was just a reminder to update version to the correct one (0.4.7 as you wrote in other comment).